lib/modem_antenna/modem_antenna.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <string.h>
+#include <nrf_modem_at.h>
+#include <modem/nrf_modem_lib.h>
+
+NRF_MODEM_LIB_ON_INIT(gnss_cfg_init_hook, on_modem_lib_init, NULL);
+
+static void on_modem_lib_init(int ret, void *ctx)
+{
+	if (ret != 0) {
+		return;
+	}
+
+	if (strlen(CONFIG_MODEM_ANTENNA_AT_MAGPIO) > 0) {
+		if (nrf_modem_at_printf("%s", CONFIG_MODEM_ANTENNA_AT_MAGPIO) != 0) {
+			printk("Failed to set MAGPIO configuration\n");
```

Comment:
Just curious: Is there a reason for not using the logger?

Reply:
Right, we should use the logger in libraries

Reply:
Somehow this didn't come to my mind. 🤦‍♂️ I'll update it.